tests/src/mods/auction_args.rs

```diff
@@ -32,35 +36,12 @@ pub struct AuctionArgsBuilder {
     minimum_bid_step: Option<U512>,
     marketplace_account: AccountHash,
     marketplace_commission: u32,
+    has_enhanced_nft: bool,
 }
 
 impl AuctionArgsBuilder {
-    pub fn new_with_necessary(
```

Comment:
This function was not used so removing it in favor of AuctionArgsBuilder::default